src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class Deadline extends Task {
+	private final LocalDate time;
+
+	/**
+	 * Deadline constructor where the boolean isDone is set to false
+	 * @param a name of Deadline
+	 * @param b date of Deadline in yyyy-mm-dd (e.g. 2021-01-31)
+	 */
```

Comment:
Perhaps there should be an empty line between description and parameter sections, as well as punctuation after each parameter description? I noticed this in your other javadoc comments too.